.cargo/config.toml

```diff
@@ -1,7 +1,2 @@
 [env]
 PCRE2_SYS_STATIC = "1"
-
-[target.aarch64-apple-darwin]
-rustflags = [
-    "-C", "link-args=-Wl,-rpath,/opt/homebrew/opt/onnxruntime/lib"
```

Comment:
That will likely be system-dependent. Should we vendor an onnxruntime?

Reply:
Also the canonical way to link something is `rustc-link-lib = ["foo"]` and `rustc-link-search = ["/path/to/foo"]`, respectively. Does that work for you? This way, we could have the former system-independently and the latter by target.

Also AFAIK this does not override existing entries (I'm not sure if your version does though).